extras/gomock.bzl

```diff
@@ -34,8 +34,15 @@ _MOCKGEN_MODEL_LIB = Label("//extras/gomock:mockgen_model")
 def _gomock_source_impl(ctx):
```

Comment:
Running buildifier on this file should fix most of the CI failures.

Reply:
Yep, that seems to have cleared up most of them!